### PR TITLE
Feature/df 479 publication

### PR DESCRIPTION
**Pull request by birom**

### CODE REVIEW

File: alv-portal-ui/translations.csv [hunk 1]
```diff
@@ -1425,3 +1425,5 @@ candidate-detail,candidate-detail.anonymous-contact.name-lastname,"Name, Vorname
...
 
+portal,portal.job-publication.publication.publicDisplay.label,"<a href=""https://www.job-room.ch"" class=""link"" target=""_blank""><i class=""fas fa-caret-right link-icon""></i>Job-Room</a> und <a href=""http://www.job-desk.ch"" class=""link"" target=""_blank""><i class=""fas fa-caret-right link-icon""></i>Job-Desk</a>","<a href=""https://www.job-room.ch"" class=""link"" target=""_blank""><i class=""fas fa-caret-right link-icon""></i>Job-Room</a> and <a href=""http://www.job-desk.ch"" class=""link"" target=""_blank""><i class=""fas fa-caret-right link-icon""></i>Job-Desk</a>","<a href=""https://www.job-room.ch"" class=""link"" target=""_blank""><i class=""fas fa-caret-right link-icon""></i>Job-Room</a> et <a href=""http://www.job-desk.ch"" class=""link"" target=""_blank""><i class=""fas fa-caret-right link-icon""></i>Job-Desk</a>","<a href=""https://www.job-room.ch"" class=""link"" target=""_blank""><i class=""fas fa-caret-right link-icon""></i>Sito Job-Room</a> e <a href=""http://www.job-desk.ch"" class=""link"" target=""_blank""><i class=""fas fa-caret-right link-icon""></i>Job-Desk</a>",,
```
[the remark this answers is not in the document]
Comment (birom) [reply alv-portal-ui/translations.csv]: fixed

File: alv-portal-ui/src/app/job-publication/job-publication-form/publication/publication.component.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<fieldset [formGroup]="publication">
+
+  <legend class="sr-only">{{'home.tools.job-publication.publication.title' | translate }}</legend>
+
+  <div class="alv-bar p-2 pb-1 mb-1">
```
Comment (99999989) [inline alv-portal-ui/src/app/job-publication/job-publication-form/publication/publication.component.html]: Please remove the class `.pb-1`, and then add the class `.narrow-form` on the checkbox below.

File: alv-portal-ui/src/app/job-publication/job-publication-form/publication/publication.component.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <alv-checkbox label="portal.job-publication.publication.publicDisplay.label"
+                  alvFormControlName="publicDisplay"></alv-checkbox>
+  </div>
+
+  <div class="alv-bar p-2 pb-1 mb-1">
```
Comment (99999989) [inline alv-portal-ui/src/app/job-publication/job-publication-form/publication/publication.component.html]: Please remove the class `.pb-1`, and then add the class `.narrow-form` on the checkbox below.

File: alv-portal-ui/src/app/job-publication/job-publication-form/publication/publication.component.html (hunk 2 above)
Comment (birom) [reply alv-portal-ui/src/app/job-publication/job-publication-form/publication/publication.component.html]: fixed

File: alv-portal-ui/src/app/job-publication/job-publication-form/publication/publication.component.html (hunk 3 above)
Comment (birom) [reply alv-portal-ui/src/app/job-publication/job-publication-form/publication/publication.component.html]: fixed

**Pull request merged**